integration-tests/deployment/ccip/rmn_test.go

```diff
@@ -0,0 +1,18 @@
+package ccipdeployment
+
+import (
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+)
+
+func TestRMN(t *testing.T) {
```

Comment:
can we move this to smoke/ccip_test.go ? It will be easier to trigger this in CI

Reply:
probably can once its complete. Won't be ready for CI for a while

Reply:
So is it unfinished? Or just unable to run in CI? Mind updating the `t.Skip()` with a more descriptive message?